# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mixer/Cargo.toml
# Commit: eaa23cb3b396
# Full Hash: eaa23cb3b39686e114b1e78a78c2f8e5c91bcb29
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2019-12-28 04:06:02
# Regressor Bug: 1605471
# File Overlap Count: 5
# Description:
#   Bug 1605471 - Update cubeb-coreaudio to acb90e9. r=alwu
#   
#   Pick commits:
#   acb90e9 - Add license for mixer crate
#   82e7ff9 - Move criterion crate to [dev-dependencies]
# ==============================================================================

diff -r 509cd3b3ec22 -r eaa23cb3b396 third_party/rust/mixer/Cargo.toml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/third_party/rust/mixer/Cargo.toml	Fri Dec 27 16:58:18 2019 +0000
@@ -0,0 +1,18 @@
+[package]
+name = "mixer"
+version = "0.1.0"
+authors = ["Chun-Min Chang <chun.m.chang@gmail.com>"]
+edition = "2018"
+license = "MPL-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+bitflags = "1.2"
+
+[dev-dependencies]
+criterion = "0.3"
+
+[[bench]]
+name = "benchmark"
+harness = false
\ No newline at end of file